### PR TITLE
[RHCLOUD-19793] Source handler tests update2

### PR DESCRIPTION
**Pull request by petracihalova**
Tests update for source handlers - part II. 
(part I. #369)

- main goal is to add/update tests to better tenancy testing
- each commit contains one updated or added test (or new helper function)
- changes for other functions in source handler will be part of next PR

handlers covered:
 - SourceList()
 - SourceGet()

**JIRA:** [RHCLOUD-19793](https://issues.redhat.com/browse/RHCLOUD-19793)

### CODE REVIEW

**Comment by petracihalova** [comment]
PR will be rebased after merge #369

**Comment by petracihalova** [comment]
rebased, ready for review

File: source_handlers_test.go [hunk 1]
```diff
@@ -913,6 +916,19 @@ func TestSourceGet(t *testing.T) {
 	if *outSrc.Name != "Source1" {
 		t.Error("ghosts infected the return")
 	}
+
+	if outSrc.ID != fmt.Sprintf("%d", sourceId) {
+		t.Errorf("source with wrong ID returned, expected %d, got %s", sourceId, outSrc.ID)
+	}
+
+	for _, src := range fixtures.TestSourceData {
```
Comment (lpichler) [inline source_handlers_test.go:931]: I am not sure about this test. You search in `fixtures.TestSourceData`. Was intention to also  add` outSrc` somehow ?

Comment (petracihalova) [reply source_handlers_test.go:931]: yes, good eye ... I changed it to use returned source

**Pull request merged**